vxquery-cli/pom.xml

```diff
@@ -61,12 +62,12 @@
                     </execution>
                 </executions>
             </plugin>
-      <!--
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-site-plugin</artifactId>
-      </plugin>
-      -->
+            <!--
```

Comment:
Done! I didn't remove that previously because it was there already in the previous CLI code.